packages/components/src/Combobox/Combobox.test.tsx

```diff
@@ -1,7 +1,127 @@
-import { cleanup } from "@testing-library/react";
+import { cleanup, render } from "@testing-library/react";
+import React from "react";
+import { Combobox } from "./Combobox";
 
 afterEach(cleanup);
 
-it("renders a Combobox", () => {
-  expect(true).toBe(true);
+describe("Combobox validation", () => {
+  it("renders without error if the correct count and composition of elements are present", () => {
+    expect(() => {
+      render(
+        <Combobox>
+          <Combobox.TriggerButton label="Button" />
+          <Combobox.Content options={[]} onSelection={jest.fn()}>
+            <></>
+          </Combobox.Content>
+        </Combobox>,
+      );
+    }).not.toThrow();
+  });
+  it("throws an error if there is no Trigger element", () => {
+    expect.assertions(1);
+    let error;
+    try {
```

Comment:
looking for feedback on this approach. it works, but I'm not sure I love it

initially I tried

```
  it("throws an error if there is no Trigger element", () => {
    expect.assertions(1);
    let error;
    const expectation = render(
      <Combobox>
        <Combobox.Content options={[]} onSelection={jest.fn()}>
          <></>
        </Combobox.Content>
      </Combobox>,
    );
    expect(expectation).toThrow(
      "Combobox must have a Trigger and Content element",
    );
  });
```

which I like significantly more. the problem is that the thrown error stops execution and we never make it to the `expect`. it complains about an ErrorBoundary which is fair, but I'd prefer not to build and implement that here since that does not appear to exist currently

Reply:
yeah, i think this should be fine